packages/gatsby-image/README.md

```diff
@@ -60,7 +60,11 @@ effect as well as lazy loading of images further down the screen.
 
 ## Install
 
-`npm install --save gatsby-image`
+`npm install --save gatsby-image intersection-observer object-fit-images`
```

Comment:
if those are in `dependencies` in package.json people don't need to install them manually

Reply:
Excellent! I wasn't sure.